digestwriter/storage.go

```diff
@@ -115,11 +114,17 @@ func (storage *DBStorage) WriteClusterInfo(cluster *ClusterName, account *Accoun
 		OrgID:         fmt.Sprint(*orgID),
 	}
 
+	logger.WithFields(logrus.Fields{
+		rowIDKey:   accountData.ID,
+		accountKey: accountData.AccountNumber,
+		orgKey:     accountData.OrgID,
+	}).Debugln("account data to insert")
+
 	tx := storage.connection.Begin()
 
 	// Insert account info in account table if not present
 	// If present, retrieve corresponding ID
-	if err = tx.Clauses(clause.OnConflict{DoNothing: true}).FirstOrCreate(&accountData, accountData).Error; err != nil {
+	if err = tx.Where(accountData).FirstOrCreate(&accountData).Error; err != nil {
```

Comment:
I agree with putting the Where() in front. But I think we need to keep the ON CONFLICT clause here to prevent errors in production deployment - when multiple instances of digestwriter will be running and two clusters from a new account/org are processed at once, conflict can happen.

Reply:
Oh! I did not think about multiple instances running at the same time =)
I'll keep it, then, and give some thinking to other possible conflicts.